srcpkgs/mangadesk/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'mangadesk'
+pkgname=mangadesk
+version=0.7.8
+revision=1
+build_style=go
+go_import_path=github.com/darylhjd/mangadesk
+short_desc="Terminal client for MangaDex"
+maintainer="Clyhtsuriva <void-packages@adjutor.xyz>"
+license="MIT"
+homepage="https://github.com/darylhjd/mangadesk"
+changelog="https://github.com/darylhjd/mangadesk/releases/"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```

Comment:
I would not use the homepage variable here and I would remove the trailing slash from the changelog url.

Reply:
Resolved with the latest commit